assets/keymaps/vim.json

```diff
@@ -371,7 +368,18 @@
       "> >": "vim::Indent",
       "< <": "vim::Outdent",
       "ctrl-pagedown": "pane::ActivateNextItem",
-      "ctrl-pageup": "pane::ActivatePrevItem"
+      "ctrl-pageup": "pane::ActivatePrevItem",
+      // tree-sitter related commands
+      "[ x": "editor::SelectLargerSyntaxNode",
```

Comment:
In visual mode you need to switch to normal mode

Reply:
Why? It works for me exactly like this